src/lumigo_tracer/wrappers/http/sync_http_wrappers.py

```diff
@@ -175,10 +175,19 @@ def _http_send_wrapper(func, instance, args, kwargs):
     with lumigo_safe_execute("add request event"):
         if headers:
             add_request_event(
-                HttpRequest(host=host, method=method, uri=uri, headers=headers, body=body)
+                HttpRequest(
+                    host=host,
+                    method=method,
+                    uri=uri,
+                    headers=headers,
+                    body=body,
+                    instance_id=id(instance),
+                )
             )
         else:
-            add_unparsed_request(HttpRequest(host=host, method=method, uri=uri, body=data))
+            add_unparsed_request(
+                HttpRequest(host=host, method=method, uri=uri, body=data, instance_id=id(instance))
```

Comment:
what is an `id` of an `instance` here?
This is an instance of an `HTTPConnection` which doesn't tell you much about the request/session/machine 🤔 
What are you trying to achieve?

Reply:
The goal is to validate the reuse of connections.
For example:
Multiple connections to the same AWS service like kinesis lambda etc.

The data is based on HTTP/S connection id + host